app/views/support_interface/settings/notify_template.html.erb

```diff
@@ -1,12 +1,21 @@
 <%= render 'settings_navigation', title: 'Send notify template' %>
 
 <h1 class="govuk-label-wrapper">
-  <label class="govuk-label govuk-label--l" for="more-detail">
+  <label class="govuk-label govuk-label--l">
     Send a Govuk notify template with attachment
   </label>
 </h1>
 
-<%= form_with url: support_interface_send_notify_template_path, method: :post do |f| %>
+<ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+  <li>Emails will only be sent once suitable and valid files have been provided.</li>
+  <li>Emails are sent in batches of 100 staggered over time, depending on the size of the distribution list.</li>
+  <li>The distribution list file must contain the header 'Email address'.</li>
+  <li>Your notify template must include the personalisation 'link_to_file'.</li>
+</ul>
+
+<%= form_with model: @form, url: support_interface_send_notify_template_path, method: :post do |f| %>
+  <%= f.govuk_error_summary %>
```

Comment:
In general, the error summary needs to be at the top of the page, above the h1. So that means the whole thing will be wrapped in the form, but that's fine.